client/src/components/App.jsx

```diff
@@ -36,7 +40,7 @@ class App extends React.Component {
 				<h1>hi from react</h1>
 				<Header />
 				<Ratings />
-				<Reviews reviewsData={this.getReviews()}/>
+				<Reviews reviewsData={this.state.reviews}/>
```

Comment:
@gorakong You should use object destructuring to pull the reviews property off of the state object.